Specification/BrAPI-Schema/BrAPI-Phenotyping/Scale.json

```diff
@@ -173,7 +167,7 @@
                 },
                 "validValues": {
                     "description": "associated valid values",
-                    "$ref": "../BrAPI-Phenotyping/Scale.json#/$defs/ValidValues",
+                    "$ref": "../BrAPI-Phenotyping/Scale.json#/$defs/ValidValue",
```

Comment:
Should be ValidValues, see my comment above, sorry for my mistake in the previous review.